src/material/core/focus-indicators/_focus-indicators.scss

```diff
@@ -1,60 +1,95 @@
 @import '../theming/theming';
 @import '../style/layout-common';
 
-/// Mixin that turns on strong focus indicators.
+/// Mixin that turns on strong focus indicators. This mixin should only be
+/// included once in your app.
+///
+/// @param {map} $config
+///   If given, customizes focus indicators with the configuration. The
+///   configuration map accepts border-style, border-color, border-width,
+///   border-radius, and margin. If a given parameter is omitted, this
+///   mixin provides a default value. More on each parameter:
+///
+///   border-style: Focus indicator line style (e.g. solid, dashed, etc).
+///   border-color: Focus indicator color.
+///   border-width: Focus indicator thickness.
+///   border-radius: Focus indicator shape.
+///   margin: Focus indicator inset/offset. Positive margin moves the focus
+///       indicator inwards, negative margin moves it outwards.
 ///
 /// @example
 ///   .my-app {
-///     @include mat-strong-focus-indicators($config);
+///     @include mat-strong-focus-indicators((
+///       border-style: solid,
+///       border-color: blue,
+///       border-width: 2px,
+///       border-radius: 4px,
+///       margin: 0px,
+///     ));
 ///   }
 @mixin mat-strong-focus-indicators($config: ()) {
   // Default focus indicator config.
   $default-config: (
     border-style: solid,
+    border-color: black,
     border-width: 3px,
     border-radius: 4px,
```

Comment:
Instead of having a default `border-radius: 4px` for all focus indicators, we could render circular focus indicators on round controls (e.g. checkbox, slide toggle, radio button, icon button, fab). This is a bit more default styles. WDYT?